packages/server-wallet/src/wallet/__test__/challenge.test.ts

```diff
@@ -34,12 +50,9 @@ it('submits a challenge when no challenge exists for a channel', async () => {
   const current = await AdjudicatorStatusModel.getAdjudicatorStatus(w.knex, channelId);
 
   expect(current.channelMode).toEqual('Open');
-  const challengeState = {
-    ...c.channelConstants,
-    ..._.pick(c.latest, ['turnNum', 'outcome', 'appData', 'isFinal']),
-  };
-  await w.challenge(challengeState);
-  expect(spy).toHaveBeenCalledWith([expect.objectContaining(challengeState)], alice().privateKey);
+
```

Comment:
nit: can you
```suggestion
    expect(c.isLedger).toBe(true)
```
for reader clarity?

Reply:
Done in b5544fe